### DISCUSSION TITLE
Min Løsning til TrafikLys

### DISCUSSION THREAD

**Discussion by Tommy-de-oz1thc** [opgaver/TrafikLys#4] [Q&A]
https://github.com/Tommy-de-oz1thc/Bogen-Om-CSharp-Ekstra-Opgaver

Du går bare ind under mappen TrafikLys.

Jeg er ikke så god til GitHub, men her er det direkte link til min løsning, der vil alle de løsninger som jeg laver komme.

Der ligger også en video hvordan det virker.

Jeg har lidt C# erfaring, men efter at have læst Bogen om C# fandt jeg ud af at der var meget jeg ikke kendte til.
Dette er min første ekstra opgave, så jeg er spændt på hvordan den bliver bedømt.

Det tog mig 2-3 timer at kode den. Jeg kan se at i forslaget til løsning har Michell brugt Enum, det er jeg ikke så stærk i, men syntes selv jeg har rimelig styr på List, så det valgte jeg at bruge. Jeg har også fået lidt hjælp af ChatGPT.

Tommy

**Comment by mcronberg** [comment]
Tak for din løsning ;)
